features/step_definitions/pos_steps.rb

```diff
@@ -18,8 +26,12 @@
       else
         cart_items << {pricing_id: p_id, quantity: 1}
       end
-      cart_update = JSON.generate({cart_items: cart_items})
+      cart_update = JSON.generate({
+        cart_items: cart_items,
+        lock_version: @cart_version
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.